clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1487,10 +1487,7 @@ describe("Fides-js TCF", () => {
             8: false,
             9: true,
           });
-          expect(tcData.purpose.legitimateInterests).to.eql({
-            [PURPOSE_2.id]: true,
-            1: false,
-          });
+          expect(tcData.purpose.legitimateInterests).to.eql({});
```

Comment:
hm is this because the cookie doesn't have purpose legitimate interests? but this used to expect something because it prioritized the experience obj?

Reply:
yes exactly, we have to ensure we're using the cookie consents instead of experience, so if it's not set there, we have to ensure it isn't set overall.